packages/models/src/crypto-assets/bitcoin/bitcoin.model.ts

```diff
@@ -1,3 +1,6 @@
+// Branded type for Bitcoin addresses
```

Comment:
Moving this type here so it can be used in other `models` but keeping the type guard in `packages/bitcoin` as it's using an external package for validation